src/test/java/com/training/api/tests/ga/GuestAccountTests.java

```diff
@@ -250,4 +250,196 @@ public void testCreateGuestAccount(){
+        request.setUidType("EMAIL");
+
+        response = new RestEngine().getResponse(map.get("gaCreateUrl"),headerMap,
+                new Gson().toJson(request));
+
+        com.training.pojos.ga.creation.Response responseElement =
+                response.as(com.training.pojos.ga.creation.Response.class);
+
+        Assertions.assertEquals(401,responseElement.getStatus(),REQUEST_UNSUCCESSFUL);
+
+        softAssertions.assertThat(responseElement.getErrors().get(0).getInternalMessage()).as(ERRORS_MESSAGE)
+                .isEqualTo("The API key header is required and should be valid.");
+
+        softAssertions.assertThat(responseElement.getErrors().get(0).getErrorCode()).as(ERROR_CODE)
+                .isEqualTo(errorCode);
+        softAssertions.assertThat(responseElement .getError().getErrorCode()).as(ERROR_CODE)
+                .isEqualTo(errorCode);
+        softAssertions.assertThat(responseElement.getError().getMessage()).as(ERRORS_MESSAGE)
+                .isEqualTo("The API key header is required and the API key provided should be valid.");
+    }
+
+    @Test
+    public void testNegativeGaCreationInvalidEmail(){
+        String firstName="Audrey";
+        String lastName="Poole";
+        String email= "testemail@@email.com";
+
+        com.training.pojos.ga.creation.Request request = new com.training.pojos.ga.creation.Request();
+        request.setBirthdate("19620802");
+        request.setEmail(email);
+        request.setFirstName(firstName);
+        request.setLastName(lastName);
+        request.setMarketingCountry("USA");
+        request.setPassword("Password1");
+        request.setPrivacyAcceptDateTime("20190524T090712GMT");
+        request.setPrivacyVersion("1.11");
+        request.setTncAcceptDateTime("20190524T090712GMT");
+        request.setTncVersion("1.8");
+        request.setSecurityQuestion("What was the first concert you attended?");
+        request.setSecurityQuestionKey("WHAT_WAS_THE_FIRST_CONCERT_YOU_ATTENDED");
+        request.setSecurityAnswer("Answer1");
+        request.setUidType("EMAIL");
+
+        response = new RestEngine().getResponse(map.get("gaCreateUrl"),headerMap,
+                new Gson().toJson(request));
+
+        com.training.pojos.ga.creation.Response responseElement =
+                response.as(com.training.pojos.ga.creation.Response.class);
+
+        Assertions.assertEquals(422, responseElement.getStatus(), RESPONSE_ERROR);
+
+        softAssertions.assertThat(responseElement.getErrors().get(0).getErrorCode()).as(ERROR_CODE)
+                .isEqualTo("GA-0103");
+
+        softAssertions.assertThat(responseElement.getErrors().get(0).getDeveloperMessage()).as(DEVELOPERS_MESSAGE)
+                .isEqualTo("The request body did not pass input validation.");
+
+        softAssertions.assertThat(responseElement.getErrors().get(0).getInternalMessage()).as(INTERNAL_MESSAGE)
+                .isEqualTo("The request body did not pass input validation.");
```

Comment:
give some different message in soft assertions for better understanding-->line 337,line 340 has same error message